src/bundle/Resources/views/themes/admin/content/tab/translations/tab.html.twig

```diff
@@ -29,9 +29,14 @@
 
         {% set body_row_cols = body_row_cols|merge([
             { content: translation.name },
-            { content: translation.languageCode },
         ]) %}
 
+        {% if ibexa_is_expert_mode() %}
```

Comment:
I recommend utilizing ```ibexa_user_settings['user_mode']``` instead of a dedicated Twig function.

If we name the function with the "expert" keyword, this designation will remain with us for the upcoming years. If we decide to change the name or introduce additional user modes, we will need to implement extra logic

Reply:
Or `ibexa_mode(...)` function if we cannot rely on user settings being present in the template.

Reply:
Or even, we can use wildcards in twig function names and do it like `ibexa_is_*_mode` and handle it that way.

Reply:
One issue with wildcards is that you cannot use mode value as a variable.

You have to use the full function name immediately.

```twig
{% set mode = 'expert' %}
{{ mode is same as 'expert' ? ibexa_is_expert_mode() : ibexa_is_smart_mode() }}
```
Using an argument does not have this limitation.

Reply:
`ibexa_user_settings['user_mode']` or `ibexa_mode(...)` is a no go for me. It's simply bad DX:

```
{% if ibexa_user_settings['user_mode'] === constant('\\Ibexa\\AdminUi\\UserSetting\\UserMode::SMART') %}
    Do something
{% endif %}
```

Wildcard function is a good idea, but we don't need it at the moment.